subsys/net/lib/wifi_credentials/Kconfig

```diff
@@ -57,4 +63,16 @@ config WIFI_CREDENTIALS_BACKEND_PSA_UID_OFFSET
 	  This library will use WIFI_CREDENTIALS_MAX_ENTRIES entries starting with the
 	  WIFI_CREDENTIALS_BACKEND_PSA_UID_OFFSET uid.
 
+
 endif # WIFI_CREDENTIALS_BACKEND_PSA
+
+config WIFI_CREDENTIALS_STATIC_SSID
```

Comment:
Just a suggestion, but perhaps it'd make sense to provide a top-level `WIFI_CREDENTIALS_STATIC` bool option to select whether static credentials are used in the first place? I think the remainig options (ssid/pass/type) could be hidden if not used.

Reply:
done